.github/scripts/go-mod-tidy-check.sh

```diff
@@ -1,27 +1,37 @@
 #!/usr/bin/env bash
-set -u
+set -eu
```

Comment:
may need to confirm this: if you `set -e` doesn't that prevent trap functionality for exit? (You may need to also add the `ERR` trap)

Reply:
The trap command will still be executed, regardless of exiting with an error or exiting cleanly — is that what you mean?

Reply:
cool, couldn't remember, wanted to double check 👍